multimodal/src/autogluon/multimodal/data/preprocess_dataframe.py

```diff
@@ -523,22 +525,30 @@ def transform_label(
         elif self.label_type == ROIS:
             y = y_df  # Do nothing. TODO: Shall we transform this?
         elif self.label_type == NER_ANNOTATION:
-            # TODO: Add transform_multimodal and process_multimodal.py
-            text_column_index = 0  # Currently, we only support one text column.
-            x_df = df[self._text_feature_names[text_column_index]]
             y = self._label_generator.transform(y_df)
-            x = self.transform_text(df)[0]
-            # Labelprocessor needs both ner annotations and text.
-            ret = {
-                NER_ANNOTATION: y,
-                TEXT: x[self._text_feature_names[text_column_index]],
-            }
-            return ret, None
         else:
             raise NotImplementedError
 
         return {self._label_column: y}, {self._label_column: self.label_type}
 
+    def transform_ner(
+        self,
+        df: pd.DataFrame,
+    ) -> Tuple[Dict[str, NDArray[(Any,), Any]], Dict[str, str]]:
```

Comment:
We can do it later but it is good to add a docstring about how we are transforming NER columns.